pkg/producer/convertor/convertor.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
Maybe we should add support for checking year of newly-added file.

Reply:
yes we should.